primitives/runtime/src/transaction_validity.rs

```diff
@@ -54,6 +54,9 @@ pub enum InvalidTransaction {
 	/// it will only be able to assume a bad signature and cannot express a more meaningful error.
 	BadProof,
 	/// The transaction birth block is ancient.
+	///
+	/// For `FRAME`-based runtimes this means:
+	/// current block number - `Era::birth` block number > `BlockHashCount`.
```

Comment:
Maybe add this in `# Possible causes` section (like with `BadProof`) and then expand on the example even more?
I think the entire formula should be in backticks, instead of just `Era::birth/BlockHashCount`.

Reply:
Just updated following these suggestions.